pennylane/transforms/optimization/single_qubit_fusion.py

```diff
@@ -82,7 +97,8 @@ def qfunc(r1, r2):
         next_gate_idx = find_next_gate(current_gate.wires, list_copy[1:])
 
         if next_gate_idx is None:
-            apply(current_gate)
+            # Apply the Rot equivalent of the gate, for consistency
+            Rot(*cumulative_angles, wires=current_gate.wires)
```

Comment:
Oh interesting --- wouldn't it be better to leave the gates as they are? E.g., CNOT rather than Rot(pi/4, 0, 0)?

Reply:
Because if the circuit is to end up on hardware, something like a CNOT will be easier to implement than Rot(pi/4, 0, 0) (even though they are equivalent)

Reply:
We could do either! I was just looking at the docs and realized I'd written "one or more gates gets fused to `Rot`", so to match I turned everything to `Rot`. But if it's easier to implement just the regular gates, I'll adjust the docstring and revert this back to `apply(current_gate)`.